crates/turbopack/src/rebase/mod.rs

```diff
@@ -50,14 +50,10 @@ impl OutputAsset for RebasedAsset {
 
     #[turbo_tasks::function]
     async fn references(&self) -> Result<Vc<OutputAssets>> {
-        let mut references = Vec::new();
-        for &module in all_referenced_modules(self.source).await?.iter() {
-            references.push(Vc::upcast(RebasedAsset::new(
-                module,
-                self.input_dir,
-                self.output_dir,
-            )));
-        }
+        let references = all_modules_iter([self.source].into_iter())
```

Comment:
This should only return the shallow references (not the whole graph). It was using `all_referenced_modules` which only returns the shallow references.

It should also include primary and secondary references.

This is used in NFT and that's why it's failing